datafusion/tests/user_defined_plan.rs

```diff
@@ -128,6 +128,8 @@ const QUERY1: &str = "SELECT * FROM sales limit 3";
 const QUERY: &str =
     "SELECT customer_id, revenue FROM sales ORDER BY revenue DESC limit 3";
 
+const QUERY2: &str = "SELECT customer_id, revenue FROM sales  UNION SELECT customer_id, revenue FROM sales ORDER BY revenue DESC limit 3";
```

Comment:
Can you please move this test into `tests/sql.rs`? I don't think `UNION` has anything to do with user defined plans.